src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private String status;
```

Comment:
An `enum` may be more appropriate because the two values we need are `added` and `deleted`.